internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -118,12 +119,28 @@ func convertSidecar(s convertSidecarOpts) ([]*template.SidecarOpts, error) {
 			DockerLabels: config.DockerLabels,
 			DependsOn:    config.DependsOn,
 			EntryPoint:   entrypoint,
+			HealthCheck:  convertContainerHealthCheck(config.HealthCheck),
 			Command:      command,
 		})
 	}
 	return sidecars, nil
 }
 
+func convertContainerHealthCheck(hc manifest.ContainerHealthCheck) *ecs.HealthCheck {
```

Comment:
I know that this is a legacy signature, but it's a bit odd that this `convert` function returns a type from the `aws-sdk` instead of `template`.

What do you think of redefining HealthCheck in `template` with the exact same signature as here?

Reply:
Agree that makes sense to me